flask/app/modules/api_v1_1/todo_resources.py

```diff
@@ -0,0 +1,62 @@
+import logging
```

Comment:
So you decided to go with the copy model?   Each version directory had a full copy of the code?